src/multi_vector_simulator/utils/constants_json_strings.py

```diff
@@ -109,6 +109,8 @@
 SOC_INITIAL = "soc_initial"
 SOC_MAX = "soc_max"
 SOC_MIN = "soc_min"
+REL_LOSSES = "rel_thermal_losses"
+ABS_LOSSES = "abs_thermal_losses"
```

Comment:
```suggestion
REL_LOSSES = "relative_thermal_losses"
ABS_LOSSES = "absolute_thermal_losses"
```
Easier for the end user to read

Reply:
Done with commit https://github.com/rl-institut/multi-vector-simulator/pull/711/commits/69c2da7838f6eed80e8f68c6d219da49d36ca7ab